File: java-store-management-system/src/Employee.java
public class Employee {
    private String firstName;
    private String lastName;
    private String SSN;

    public Employee() {
    }

    public Employee(String firstName, String lastName, String SSN) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.SSN = SSN;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getSSN() {
        return SSN;
    }

    public void setSSN(String SSN) {
        this.SSN = SSN;
    }

    public void printInformation() {
        String message = String.format("Information on %s %s\nSSN: %s",
                firstName, lastName, SSN);
        System.out.println(message);
    }
}

File: python-store-management-system/Employee.py
class Employee:
    def __init__(self, first_name, last_name, SSN):
        self.first_name = first_name
        self.last_name = last_name
        self.SSN = SSN

    def __str__(self):
        return f"Employee: {self.first_name}, {self.last_name}, {self.SSN}"
